website/scripts/website-build.sh

```diff
@@ -0,0 +1,39 @@
+# Repo which we are cloning and executing npm run build:deploy-preview within
```

Comment:
Will the web team be managing updates to this file? Either which way, it'd be great if there was a lead-in comment explaining what the script is and who owns it. 👍

Reply:
The web team will manage this script. I'll add a comment!